suite-native/app/app.config.ts

```diff
@@ -75,11 +75,18 @@ const getPlugins = (): ExpoPlugins => {
             },
         ],
         [
-            'expo-barcode-scanner',
+            'expo-camera',
```

Comment:
Maybe we should also add `"recordAudioAndroid": false` because default value seems to be `true`? https://docs.expo.dev/versions/latest/sdk/camera/#configurable-properties

Reply:
good point 👍

Reply:
Maybe, can you, please, remove the old one and add the new library to this list?
 https://github.com/trezor/trezor-suite/blob/a35125abeee5c78d9e7f284b0f9bd80ca6ff7765/scripts/list-outdated-dependencies/mobile-dependencies.txt#L35


And I see, there is also `netInfo` missing from the list, can you please add this as well? 🙏

Reply:
@matejkriz permission fixed in 22fe1f4715.

@vytick the QR code-related libraries were already added there with the original commit. Since netinfo is unrelated to this issue, I created a separate PR for it: https://github.com/trezor/trezor-suite/pull/14222

Reply:
Sorry, I must be blind 🙈 . And thanks!  🙏